wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/ConversationTableViewDataSource.swift

```diff
@@ -105,6 +105,8 @@ final class ConversationTableViewDataSource: NSObject {
 
     private(set) var currentSections: [Section] = []
 
+    private var isChatBubbleSimpleEnabled: Bool = false
```

Comment:
issue: Maybe I'm missing something but I don't think this needs to be var or have a value set.

```suggestion
    private let isChatBubbleSimpleEnabled: Bool
```

Reply:
Done